### ISSUE TITLE
Image keyboard nav unclear and maybe actually pointless

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4867]
When tabbing to an image, for example in the model list, or when opening a model and tabbing to the image, the keyboard focus is not very visible.

Also on the detail page of a model, after tabbing to the image, nothing happens when you press enter, leaving users wondering why you can tab to the image in the first place if they cannot interact with it.
